### DISCUSSION TITLE
OpenAPI Example with multipart/form-data not showing up

### DISCUSSION THREAD

**Discussion by SeeRich** [fastapi/fastapi#10937] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Annotated
from fastapi import FastAPI, UploadFile, Form, File

# Run using uvicorn main:app --reload
# Visit: http://127.0.0.1:8000/docs

app = FastAPI()

@app.post("/form-test")
async def form_test(
        str_data: Annotated[str, Form(examples=["HELLO"])],
        file: Annotated[UploadFile, File()]):
    return
```

### Description

The `HELLO` example is not shown in the docs.

![fastapi](https://github.com/tiangolo/fastapi/assets/33664011/6b4deec4-c7bf-4d43-970c-25b30eacc376)

### Operating System

Windows

### FastAPI Version

0.108.0

### Pydantic Version

2.5.3

### Python Version

3.11.3

**Comment by Kludex** [comment, marked as answer]
I've created https://github.com/tiangolo/fastapi/issues/10999 from this discussion.
